modules/networking/main.tf

```diff
@@ -7,12 +7,13 @@ resource "google_compute_network" "this" {
   delete_default_routes_on_create = true
 }
 
-// Create a default route to the Internet.
-resource "google_compute_route" "egress-inet" {
+// Allow private Google access from the VPC.
+resource "google_compute_route" "private-google-access" {
```

Comment:
Add a `moved` for this to avoid the recreation?

Reply:
Done